test/src/test.coffee

```diff
@@ -25,6 +27,9 @@ test "summary is focusable", (assert) ->
   done = assert.async()
   summary = getElement("summary")
   defer ->
+    if navigator.userAgent.match(/Trident|Edge/)
```

Comment:
Can we test for native support instead of user agent sniffing here? That way the tests won't break if/when Edge implements the `<details>` element. Maybe `if (typeof HTMLDetailsElement is "undefined")`?